test/OrchardCore.Tests/DisplayManagement/Decriptors/LocationParserTests.cs

```diff
@@ -1,66 +1,101 @@
-﻿using System.Linq;
+using System.Linq;
 using OrchardCore.DisplayManagement.Descriptors;
 using Xunit;
 
 namespace OrchardCore.Tests.DisplayManagement.Decriptors
 {
     public class LocationParserTests
     {
-        [Fact]
-        public void ZoneShouldBeParsed()
+        [Theory]
+        [InlineData("Content")]
+        [InlineData("Content:5")]
+        [InlineData("Content:5#Tab1")]
+        [InlineData("Content:5@Group1")]
+        [InlineData("Content:5@Group1#Tab1")]
+        [InlineData("/Content")]
+        [InlineData("/Content:5")]
+        [InlineData("/Content:5#Tab1")]
+        [InlineData("/Content:5@Group1")]
+        [InlineData("/Content:5@Group1#Tab1")]
+        public void ZoneShouldBeParsed(string location)
         {
-            Assert.Equal("Content", new PlacementInfo { Location = "/Content" }.GetZones().FirstOrDefault());
-            Assert.Equal("Content", new PlacementInfo { Location = "Content" }.GetZones().FirstOrDefault());
-            Assert.Equal("Content", new PlacementInfo { Location = "Content:5" }.GetZones().FirstOrDefault());
-            Assert.Equal("Content", new PlacementInfo { Location = "Content:5#Tab1" }.GetZones().FirstOrDefault());
-            Assert.Equal("Content", new PlacementInfo { Location = "Content:5@Group1" }.GetZones().FirstOrDefault());
-            Assert.Equal("Content", new PlacementInfo { Location = "Content:5@Group1#Tab1" }.GetZones().FirstOrDefault());
```

Comment:
Refactored to avoid vague failure messages (e.g. `Expected "Content" but was "SomethingElse")